kubernetes/resource_kubernetes_job_test.go

```diff
@@ -119,28 +200,36 @@ func TestAccKubernetesJob_ttl_seconds_after_finished(t *testing.T) {
 		IDRefreshName: "kubernetes_job.test",
 		IDRefreshIgnore: []string{
 			"spec.0.selector.0.match_expressions.#",
-			"spec.0.selector.0.match_labels.%",
-			"spec.0.template.0.spec.0.container.0.resources.0.limits.#",
-			"spec.0.template.0.spec.0.container.0.resources.0.requests.#",
 		},
 		ProviderFactories: testAccProviderFactories,
 		CheckDestroy:      testAccCheckKubernetesJobDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccKubernetesJobConfig_ttl_seconds_after_finished(name),
+				Config: testAccKubernetesJobConfig_ttl_seconds_after_finished(name, busyboxImageVersion),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					testAccCheckKubernetesJobExists("kubernetes_job.test", &conf),
-					// FIXME uncomment this check when the TTLSecondsAfterFinished feature gate defaults to true
-					// resource.TestCheckResourceAttr("kubernetes_job.test", "spec.0.ttl_seconds_after_finished", "60"),
+					resource.TestCheckResourceAttr("kubernetes_job.test", "spec.0.ttl_seconds_after_finished", "60"),
 				),
-
-				// FIXME remove this when TTLSecondsAfterFinished defaults to true
-				ExpectNonEmptyPlan: true,
 			},
```

Comment:
I didn't actually mean to remove this comment... but then after it was removed, I figured I'd like the tests to keep telling me that this field isn't working yet. It's kind of noise, but I'd rather it be noisy than have to remember to come back and fix it later. Also, I learned that minikube can actually test this!

```
minikube start --feature-gates=TTLAfterFinished=true
```